.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+| Q             | A
+| ------------- | ---
+| Is bugfix?    | ✔️/❌
+| New feature?  | ✔️/❌
+| Breaks BC?    | ✔️/❌
+| Tests pass?   | ✔️/❌
```

Comment:
This line was excluded in recent yiisoft/template.

Reply:
fixed.